server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/Method.java

```diff
@@ -123,7 +127,15 @@ public String toString() {
     Request.Name requestHandled();
 
     Class<? extends JmapRequest> requestType();
-    
-    Stream<JmapResponse> process(JmapRequest request, MethodCallId methodCallId, MailboxSession mailboxSession);
 
+    default Flux<JmapResponse> process(JmapRequest request, MethodCallId methodCallId, MailboxSession mailboxSession) {
+        return Mono.fromCallable(() -> processToStream(request, methodCallId, mailboxSession))
+            .flatMapMany(Flux::fromStream)
+            .subscribeOn(Schedulers.elastic());
```

Comment:
why?

Reply:
Otherwize I got a blocking called in a //  netty thread ;-) (toStream is blocking)

Reply:
why not put that line where you call `toStream`?